samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs

```diff
@@ -936,6 +941,45 @@ private void ShowPopupGravityUi()
             }
         }
 
+        private void ShowPermissionsUi()
+        {
+            DrawStatus();
+            DrawTitle("PopupGravity Ui");
+
+            if (GUI.Button(CalcGrid(0, 1), "Has Drive - AppData"))
+            {
+                Status = "Drive - AppData Permission " + PlayGamesPlatform.Instance.HasPermission(
+                             /* scope= */ "https://www.googleapis.com/auth/drive.appdata");
+            }
+            else if (GUI.Button(CalcGrid(1, 1), "Has Games Lite"))
+            {
```

Comment:
We can ask for email permission as we can check if it works fine in info page

Reply:
This made me realize a bug I have. Fixed that one too while addressing this comment. Addressed the comment and fixed the bug with the change below in AndroidTokenClient

```
  account = accountWithNewScopes;
  email = account.Call<string>("getEmail");
  idToken = account.Call<string>("getIdToken");
  authCode = account.Call<string>("getServerAuthCode");
```